README.md

```diff
@@ -144,10 +144,12 @@ AGNOS is the operating system used in C3X. The latest one is needed to run TSK M
 
 1-2. Connect C3X to your Wi-Fi network.
 
-1-3. Install `Custom Software` with URL `commaai/master-ci`
+1-3. Don't choose `Install openpilot`. Instead, choose `Custom Software` with URL `commaai/master-ci`
```

Comment:
really dig the negative here. Don't do this is perfect!

Reply:
I saw two people do this 😅